connect/api/connect.api

```diff
@@ -284,14 +284,6 @@ public final class com/stripe/android/connect/appearance/fonts/CustomFontSource$
 	public final fun serializer ()Lkotlinx/serialization/KSerializer;
 }
 
-public final class com/stripe/android/connect/databinding/StripeConnectWebviewBinding : androidx/viewbinding/ViewBinding {
```

Comment:
👀 This XML layout was used by `StripeConnectWebViewContainer`, but now it constructs the view programmatically now that it's getting the WebView from the ViewModel.

Reply:
This was only used internally and was not meant to be exposed in the public API.